tests/cases/standalone/common/function/json/json.sql

```diff
@@ -6,3 +6,19 @@ SELECT json_path_exists(parse_json('{"a": 1, "b": 2}'), '$.c');
 SELECT json_path_exists(parse_json('[1, 2]'), '[0]');
 
 SELECT json_path_exists(parse_json('[1, 2]'), '[2]');
+
+SELECT json_path_exists(parse_json('[1, 2]'), NULL);
+
+-- SELECT json_path_exists(NULL, '$.a');
```

Comment:
I would like to fix [this](#4865) in a follow-on PR.

Reply:
The JSON null is not the same as the null in a datatype system. 
```sql
SELECT json_path_exists(parse_json('null'), '$.a');
```

Reply:
> The JSON null is not the same as the null in a datatype system.
> `SELECT json_path_exists(parse_json('null'), '$.a');` 

Thx for pointing that out. I wonder how it should behave in two different cases?
1. Given case like `json_path_exists(NULL, '$.a')`, leave it unsupported or return NULL (as https://docs.databend.com/sql/sql-functions/semi-structured-functions/json-path-exists)?
2. When given the json string is "null", return NULL?

Please let me know the preferred behaviour so that I can fix potential bias.

Reply:
let's throw a unsupported error